mon-pix/tests/unit/routes/competences/resume-test.js

```diff
@@ -40,32 +34,15 @@ describe('Unit | Route | Competence | Resume', function() {
 
   describe('#model', function() {
 
-    it('should create the CompetenceEvaluation if it not exits', function() {
-      // given
-      peekAllStub.returns([competenceEvaluationFromAnotherCompetence]);
-
-      const params = { competence_id: competenceId };
-
-      // when
-      const promise = route.model(params);
-
-      // then
-      promise.then((resultModel) => {
-        expect(resultModel).to.equal(competenceEvaluationCreated);
-      });
-
-    });
-
-    it('should return the CompetenceEvaluation in store if it exists', function() {
+    it('should create the CompetenceEvaluation if it not exits', async function() {
```

Comment:
`if it not` => `if it does not`
`exits` => `exists`

Reply:
En fait j'ai l'impression que le titre du test n'est plus bon, puisqu'on ne fait plus de création ?